### PR TITLE
Update kube_node_status_allocatable unit doc

### PR DESCRIPTION
**Pull request by jumbosushi**
**What this PR does / why we need it**:

Documentation for `kube_node_status_allocatable` unit column is sparse. This completes it. 

CleanShot 2022-06-16 at 17 28 26@2x

**How does this change affect the cardinality of KSM**: *(increases, decreases or does not change cardinality)*

Does not change cardinality

### CODE REVIEW

File: docs/node-metrics.md [hunk 1]
```diff
@@ -9,6 +9,6 @@
...
 | kube_node_status_capacity | Gauge | The capacity for different resources of a node | `cpu`=&lt;core&gt; <br> `ephemeral_storage`=&lt;byte&gt; <br> `pods`=&lt;integer&gt; <br> `attachable_volumes_*`=&lt;byte&gt; <br> `hugepages_*`=&lt;byte&gt; <br> `memory`=&lt;byte&gt; |`node`=&lt;node-address&gt; <br> `resource`=&lt;resource-name&gt; <br> `unit`=&lt;resource-unit&gt;| STABLE |
-| kube_node_status_allocatable | Gauge | The allocatable for different resources of a node that are available for scheduling | byte |`node`=&lt;node-address&gt; <br> `resource`=&lt;resource-name&gt; <br> `unit`=&lt;resource-unit&gt;| STABLE |
+| kube_node_status_allocatable | Gauge | The allocatable for different resources of a node that are available for scheduling | `cpu`=&lt;core&gt; <br> `ephemeral_storage`=&lt;byte&gt; <br> `pods`=&lt;integer&gt; <br> `attachable_volumes_*`=&lt;byte&gt; <br> `hugepages_*`=&lt;byte&gt; <br> `memory`=&lt;byte&gt; |`node`=&lt;node-address&gt; <br> `resource`=&lt;resource-name&gt; <br> `unit`=&lt;resource-unit&gt;| STABLE |
```
Comment (fpetkovski) [inline docs/node-metrics.md:12]: Is attachable_volumes a byte metric?

Comment (jumbosushi) [reply docs/node-metrics.md:12]: I believe so based on the following code. It also matches with `attachable_volumes` unit from `kube_node_status_capacity`:
https://github.com/kubernetes/kube-state-metrics/blob/10dde2115c09635aa415f6db8df7cbfd2466b7c1/internal/store/node.go#L294-L301

Comment (fpetkovski) [reply docs/node-metrics.md:12]: That's really interesting, thanks.

**Comment by fpetkovski** [comment]
/lgtm

Thanks for making the docs clearer 👍

**Pull request merged**